xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/MailSenderScriptService.java

```diff
@@ -282,6 +301,17 @@ public MailSenderConfiguration getConfiguration()
         return this.senderConfiguration;
     }
 
+    /**
+     * @return the general configuration about mails; i.e. everything that did not fit elswhere
+     * @since 11.1
+     * @since 10.11.4
+     */
+    public MailGeneralConfiguration getGeneralConfiguration()
```

Comment:
That doesn't look good to me. We're in the mail sender and there's no reason for it to know the general config.

Reply:
The place would be in https://github.com/xwiki/xwiki-platform/blob/fb8e06afa1a7e46e33ce311eb4b09be81b0567f6/xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-general/src/main/java/org/xwiki/mail/script/GeneralMailScriptService.java#L39 but I need to check below why it would be needed.